xml/Global/BottomBar.xml

```diff
@@ -1,20 +1,28 @@
 <Defaults>
   <Button class="navbar"
+    tooltipOffset="-135"
```

Comment:
Looks like this is too far away (if necessary at all?) compared to the position of the buttons.